src/multisketch.rs

```diff
@@ -166,3 +263,37 @@ impl fmt::Display for MultiSketch {
         Ok(())
     }
 }
+
+impl PartialEq for MultiSketch {
+    fn eq(&self, other: &Self) -> bool {
+        if self.sketch_metadata.len() != other.sketch_metadata.len() {
+            panic!("sketch_metadata length mismatch: self {} != other {}", 
+                   self.sketch_metadata.len(), other.sketch_metadata.len());
+        }
+        for (i, (self_sketch, other_sketch)) in self.sketch_metadata.iter().zip(other.sketch_metadata.iter()).enumerate() {
+            if self_sketch != other_sketch {
+                panic!("Mismatch in sketch_metadata at index {}: {:?} != {:?}", 
+                       i, self_sketch, other_sketch);
+            }
+        }
```

Comment:
These should be the same for all sketches in a multisketch, so I think this check may not be needed